src/lib/invoiceParser.ts

```
@@ -0,0 +1,211 @@
+      const supplierText = supplierElement?.textContent?.replace(/\s+/g, ' ').trim()
+      const supplier = supplierText?.replace(/^Supplied by:\s*/, '').trim()
+      const priceText = priceElement?.textContent?.trim()
+      if (!priceText) {
+        throw new InvoiceParserError('Item price not found')
+      }
+      const itemPrice = InvoiceParser.parseAmountToMilliDollars(priceText)
+      const quantityElement = itemElement.querySelector('.od-item-view-qty span')
+      const quantityText = quantityElement?.textContent?.trim()
+      let quantity = 1
+      if (quantityText && quantityText !== '') {
+        quantity = parseInt(quantityText, 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)